src/components/sidebar-subheader/SidebarSubheader.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Button } from '../../components/button';
+import { IconChevronLeft } from '../../icons/IconChevronLeft';
+import { SButtonOverwrite } from './styles/SButtonOverwrite';
+import { SSidebarSubheader } from './styles/SSidebarSubheader';
+import { SSidebarSubheaderText } from './styles/SSidebarSubheaderText';
+
+interface ISidebarSubheader {
+  subheaderText?: string,
+  onExit?: () => void,
```

Comment:
@steiningermona this is really good. But I would change to onClick since onExit is used for other stuff and might by confusion. But very good anyways!!